<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-6P3KT7T9" failures="1" tests="1" name="Test(failed)" time="128.428" errors="0" timestamp="2024-09-19T07:24:41 IST">
  <testcase classname="testSuite.TestCase01" name="tc01" time="128.423">
    <failure type="java.lang.AssertionError" message="Failed message is displayed!...
	Test is faild due to this,
	Test is faild due to this,
	Element is not visible!...,
	Element is not visible!...">
      <![CDATA[java.lang.AssertionError: Failed message is displayed!...
Test is faild due to this,
Test is faild due to this,
Element is not visible!...,
Element is not visible!...
at utils.AssertUtils.softAssertAll(AssertUtils.java:20)
at testSuite.TestCase01.tc01(TestCase01.java:36)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- tc01 -->
</testsuite> <!-- Test(failed) -->
